packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -1,30 +1,34 @@
 import React from 'react';
 import { localize } from '@deriv/translations';
-import { Text } from '@deriv/components';
+import { storeSetting, getSetting } from '../../utils/settings';
+import { getUrlBase } from '@deriv/shared';
+import TourGuide from './tour-guide';
+import { CallBackProps } from 'react-joyride';
 
-type TJoyrideConfig = Record<'showProgress' | 'showSkipButton' | 'spotlightClicks' | 'disableBeacon', boolean>;
+export const getImageLocation = (image_name: string) => getUrlBase(`/public/images/common/${image_name}`);
+
+export const setJoyRideToken = () => {
+    return storeSetting('joyride_trigger', new Date().getTime());
+};
 
 type TStep = {
-    label: string | boolean;
+    label: string;
     content: string;
 };
+export const getJoyrideToken = () => {
+    return getSetting('joyride_trigger');
+};
+
+export const handleJoyrideCallback = (data: CallBackProps) => {
+    const { action, index, status, type } = data;
+
+    const getJoyrideTokenSettings = getJoyrideToken();
```

Comment:
we can make thi line shorter:
`if(!getJoyrideToken()) setJoyRideToken();`

Reply:
done ✅